NDB.Covid19/NDB.Covid19.iOS/Views/Welcome/Welcome.storyboard

```diff
@@ -46,7 +46,7 @@
                                     <constraint firstAttribute="height" constant="48" id="nMy-oM-zWq"/>
                                 </constraints>
                             </stackView>
-                            <activityIndicatorView opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" hidesWhenStopped="YES" style="gray" translatesAutoresizingMaskIntoConstraints="NO" id="3032">
+                            <activityIndicatorView hidden="YES" opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" hidesWhenStopped="YES" style="gray" translatesAutoresizingMaskIntoConstraints="NO" id="3032">
```

Comment:
why there are changes in this file?

Reply:
Warning: "Hides when stopped but is neither hidden nor animating"

Reply:
ok